cs41_final.sh

```diff
@@ -59,6 +83,91 @@ good_bye(){
   echo -e "${MENU}***************** ${RED_TEXT}GOOD BYE ${MENU}******************${NORMAL}"
   echo -e "${MENU}*********************************************${NORMAL}"
 }
+
+#----- Option 1 -----
+currentUsers(){
+  echo "Currently Logged-In Users:"
+  users
+}
+
+#----- Option 2 -----
+listActiveProcess(){
+  echo "List of active processes for user: $userName"; 
+  ps -u $userName
+}
+
+#----- Option 3 -----
+monitorActiveProcess(){
+  echo "Monitoring of active processes for user: $userName";
+  echo -e "${RED_TEXT}*****ctrl-c to return to menu*****";
```

Comment:
when you run top, you can't see any of this text.  
You probably want to put in something like: 

echo "Press any key to continue to top."
read #pause until a key is pressed to read our text